diff -Nur -x '*.orig' -x '*~' notify-osd-0.9.7/data/Makefile.in notify-osd-0.9.7.new/data/Makefile.in
--- notify-osd-0.9.7/data/Makefile.in	2009-03-27 21:01:28.000000000 +0100
+++ notify-osd-0.9.7.new/data/Makefile.in	2009-03-30 09:53:36.000000000 +0200
@@ -199,7 +199,7 @@
 top_srcdir = @top_srcdir@
 SUBDIRS = icons
 servicedir = /usr/share/dbus-1/services
-service_DATA = org.freedesktop.Notifications.service.in
+service_DATA = org.freedesktop.Notifications.service
 EXTRA_DIST = $(service_DATA)
 all: all-recursive
 
